deny.toml

```diff
@@ -105,6 +96,13 @@ reason = """
 `PyString::from_object` is not used in the codebase.
 """
 
+[[advisories.ignore]]
+id = "RUSTSEC-2025-0363"
```

Comment:
Is it by pure chance that the ID of the removed item is so similar to the added one ?
`RUSTSEC-2024-0363` vs `RUSTSEC-2025-0363`

Reply:
Oups https://rustsec.org/advisories/RUSTSEC-2025-0026